README.org

```diff
@@ -114,6 +114,11 @@ If you want custom behavior, such as
 
 GPTel provides a general =gptel-request= function that accepts a custom prompt and a callback to act on the response. You can use this to build custom workflows not supported by =gptel-send=.  See the documentation of =gptel-request=, and the [[https://github.com/karthink/gptel/wiki][wiki]] for examples.
 
+** Additional Configuration
+
+- You can override the OpenAI API endpoint by customizing ~gptel-openai-endpoint~.
```

Comment:
Replace with `=gptel-openai-endpoint=`, as I use `=` delimiters for verbatim commands and `~` delimiters for notating keybindings. (There is no distinction between them in the Github renderer but these are different things when viewing this file in Emacs.)